libs/order/driver/magento/2.4.1/src/queries/fragments/order-item.ts

```diff
@@ -0,0 +1,58 @@
+import gql from 'graphql-tag';
+
+export const orderBundleItemSelectedOptionFragment = gql`
+  fragment orderBundleItemSelectedOption on ItemSelectedBundleOption {
+    label
+    values {
+      product_name
+    }
+  }
+`;
+
+export const orderItemFragment = gql`
+  fragment orderItem on OrderItem {
+    quantity_ordered
+    quantity_canceled
+    quantity_shipped
+    quantity_invoiced
+    product_url_key
+    product_sku
+    product_name
+    product_sale_price {
+      value
+    }
+    discounts {
+      amount {
+        value
+      }
+    }
+    selected_options {
+      label
+      value
+    }
+  }
+`;
+
+export const orderBundleItemFragment = gql`
+  fragment orderBundleItem on BundleOrderItem {
```

Comment:
I think this could simply be:
```
fragment orderBundleItem on BundleOrderItem {
    bundle_options {
      ...orderBundleItemSelectedOption
    }
  }
  ${orderBundleItemSelectedOptionFragment}
}
```
Then the usage would be like:
```
export const orderShipmentBundleItemFragment = gql`
  fragment orderShipmentBundleItem on BundleShipmentItem {
    quantity_shipped
    order_item {
      ...orderBundleItem,
      ...orderItem
    }
  }
  ${orderBundleItemFragment}
  ${orderItemFragment}
`;
```

Reply:
yeah that's better. changed